Services/AuthenticationService/Validators/UserEmailPasswordValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DTO;
+using FluentValidation;
+
+namespace AuthenticationService.Validators
```

Comment:
Do we have to have this stuff only in authentification service? 
Why we don't move this to common validation? For example , we have to check email and password in registration also

Reply:
The task item was marked with authentication service, but i get your point, will think about it

Reply:
You can also suggest where you think this validator needs to be located

Reply:
I think it would be okay in common. What if we create folder fluentvalidation in project CommonValidation

Reply:
Do we have a CommonValidation project? 0_0
As I can see, we have project Kernel with CommonValidation.cs class

Reply:
Oh, I missed that. Okay, we can create in Kernel folder with Validation